scripts/pre_commit_linter.py

```diff
@@ -423,6 +424,8 @@ def _lint_css_files(
             stdout.put(linter_stdout)
 
     if num_files_with_errors:
+        for error in stdout_list:
```

Comment:
Hmm, `stdout_list` isn't going to have anything in there. I see the initialization, but not the additions to that list.